packages/mandelbrot/src/theme.js

```diff
@@ -87,6 +87,24 @@ module.exports = function (options) {
         .filter((url) => url)
         .map((url) => (url === 'default' ? `/${config.static.mount}/css/highlight.css` : url));
 
+    config.information = (
```

Comment:
Shouldn’t we explicitly check for `undefined` so users can set `null` or `false` to disable the default in addition to `[]`?

Reply:
Nah I don't think so, the most correct (also perf friendly) way to disable the information section would be to remove it from `config.nav`. This is also similar to how other options behave, config.nav and config.panels, for example.